Dockerfile

```diff
@@ -135,6 +135,10 @@ COPY --from=builder /home/aeternity/node/local ./local
 COPY ./docker/aeternity.yaml /home/aeternity/aeternity.yaml
 COPY ./docker/aeternity-dev.yaml /home/aeternity/aeternity-dev.yaml
 COPY ./docker/healthcheck.sh /home/aeternity/healthcheck.sh
+
+COPY --from=aeternity/aeternity:v7.0.0-bundle /usr/local/lib/librocksdb.so.7.10.2 /usr/local/lib/
+
+RUN ln -fs librocksdb.so.7.10.2 /usr/local/lib/librocksdb.so.7.10     && ln -fs librocksdb.so.7.10.2 /usr/local/lib/librocksdb.so.7     && ln -fs librocksdb.so.7.10.2 /usr/local/lib/librocksdb.so     && ldconfig
```

Comment:
please reformat to multiline and remove the extra spaces i.e. https://github.com/aeternity/aeternity/blob/master/Dockerfile#L24-L28